packages/calypso-products/src/types.ts

```diff
@@ -130,6 +130,8 @@ export type JetpackProductCategory = ( typeof JETPACK_PRODUCT_CATEGORIES )[ numb
 
 // All
 export type ProductSlug = WPComProductSlug | JetpackProductSlug;
+
+// TODO: Remove this in favor of the auto-generated plan slugs in plans-list?
 export type PlanSlug = WPComPlanSlug | JetpackPlanSlug;
```

Comment:
What's the difference between this and the keys of `PLANS_LIST`?

Reply:
I'm not sure. It was added by https://github.com/Automattic/wp-calypso/pull/52836 so maybe a clue is there, but a lot of the types in this package are messy since it was a big dynamic mess of data before I dumped it in this package.

Reply:
Cool, I'll check that out!